### PR TITLE
present conjugator function

### PR DESCRIPTION
**Pull request by dsope05**

### CODE REVIEW

File: present_conjugator.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+//Left breakdown function in here so that the conjugate() function is easier to test.
+const breakdown = (input) => {
```
Comment (VigneshRavichandran02) [inline present_conjugator.js:2]: Please import 'breakdown' function from 'breakdown.js' once it gets merged in.

File: present_conjugator.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+//Left console.logs in here so that testing the conjugation is easy. Return conjugate and remove console.logs when testing is complete.
+function conjugate(word){
+
+let wordLength = word.length;
+let conjugate = '';
+
+if (word[wordLength-2] === '하'){
+  conjugate = word.slice(0, wordLength-2);
+  conjugate = conjugate.concat('해');
+  console.log('shoked', conjugate);
```
Comment (VigneshRavichandran02) [inline present_conjugator.js:32]: remove 'console.log'

File: present_conjugator.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  return;
```
Comment (VigneshRavichandran02) [inline present_conjugator.js:33]: the function returns undefined. Please fix this

File: present_conjugator.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+let brokeWord = breakdown(word[wordLength-2]);
+let brokeLength = brokeWord.length;
+
+if(brokeWord[brokeLength-1] === 0){
+  conjugate = word.slice(0, wordLength-1);
+  console.log('broke', conjugate);
```
Comment (VigneshRavichandran02) [inline present_conjugator.js:41]: same as above

File: present_conjugator.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  return;
```
Comment (VigneshRavichandran02) [inline present_conjugator.js:42]: same as mentioned before

File: present_conjugator.js [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+if(brokeWord[brokeLength-1] === 8){
+  conjugate = word.slice(0, wordLength-1);
+  conjugate = conjugate.concat('아');
+console.log('yoked', conjugate);
```
Comment (VigneshRavichandran02) [inline present_conjugator.js:48]: same as above

File: present_conjugator.js [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+return;
```
Comment (VigneshRavichandran02) [inline present_conjugator.js:49]: same as above

File: present_conjugator.js [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+conjugate = word.slice(0, wordLength-1);
+conjugate = conjugate.concat('어');
+console.log('smoked', conjugate);
```
Comment (VigneshRavichandran02) [inline present_conjugator.js:54]: same as above

File: present_conjugator.js [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+return;
```
Comment (VigneshRavichandran02) [inline present_conjugator.js:55]: same as above

File: present_conjugator.js [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+conjugate('주어다');
```
Comment (VigneshRavichandran02) [inline present_conjugator.js:58]: Please remove any test code before submitting a pull request

File: presentconjugate.js [hunk 11]
```diff
@@ -0,0 +1,69 @@
+
```
Comment (VigneshRavichandran02) [inline presentconjugate.js:1]: Could you please explain the purpose of this file. It look like it does the same functions as "present_conjugator.js"

File: present_conjugator.js [hunk 12]
```diff
@@ -0,0 +1,58 @@
+//Left breakdown function in here so that the conjugate() function is easier to test.
```
Comment (VigneshRavichandran02) [inline present_conjugator.js:1]: Please use Camel notation for file names

File: presentconjugate.js [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+   let medialValue = Math.floor(total/28);
+   letterArray.push(medialValue);
+   let finalValue = total - (28*medialValue);
+   if (finalValue > 0) {
+     letterArray.push(finalValue);
+   }
+   return letterArray;
+ }
+
+const reverseBreakdown = (input) => {
```
Comment (VigneshRavichandran02) [inline presentconjugate.js:22]: Please use "recomposition.js"

File: presentconjugate.js [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+conjugate = word.slice(0, wordLength-1);
+conjugate = conjugate.concat('어');
+console.log('smoked', conjugate);
+return;
+}
+
+//conjugate('주어다');
+reverseBreakdown([0, 8]);
```
Comment (VigneshRavichandran02) [inline presentconjugate.js:69]: test code - remove

File: present_conjugator.js (hunk 2 above)
Comment (dsope05) [inline present_conjugator.js]: i left console.logs in here so that the function can be tested more thoroughly for correct korean conjucation

**Pull request merged**